Makefile.common

```diff
@@ -24,6 +24,9 @@ CFLAGS :=
 LDFLAGS :=
 LDFLAGS_PREFIX :=
 
+# filter any other flags passed to OLVL than -O
```

Comment:
maybe add comment that You can set specific "-Ox" globally for project by setting `OLVL` variable (so usability description)?